### PR TITLE
feat(native components): remove native components

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  

- Moves all native component css to dir tegel/native-components-DEPRECATED
- Updated tsconfig
- Corrected the name of the spinner story name
- Updated data-table and dropdown to use sdds-checkbox instead of native component.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1470

**How to test**  
1. Go to storybook link below
2. Check that all the web components are still working.

**Suggested test steps**
- [ ] Browser testing (Chrome, Safari, Firefox) 
- [ ] Keyboard operability
- [ ] Interactive elements have labels.
- [ ] Storybook controls
- [ ] Design/controls/props is aligned with other components 
- [ ] Dark/light mode and variants 
- [ ] Input fields – values should be displayed properly 
- [ ] Events

### CODE REVIEW

**Comment by mJarsater** [comment]
Lets not do this one, since we are moving repos instead.

**Comment by mJarsater** [comment]
Will close this due to repository move.

File: tegel/src/global/global.scss [hunk 1]
```diff
@@ -27,58 +27,47 @@
...
 @import 'src/components/modal/modal';
 @import 'src/components/divider/divider-vars';
 @import 'src/components/popover-canvas/popover-canvas-vars';
-@import 'src/components/popover-menu/popover-menu-item';
 @import 'src/components/popover-menu/popover-menu-vars';
-@import 'src/components/spinner/spinner';
+@import 'src/components/radio-button/radio-button-vars';
+@import 'src/components/side-menu/side-menu-vars';
+
+// TODO - Remove the webcomponent folder
```
Comment (timrombergjakobsson) [inline tegel/src/global/global.scss:59]: Do we need this comment?

**Review by nathalielindqvist — APPROVED** [review]
LGTM! 🚀

**Review by timrombergjakobsson — APPROVED** [review]

**Pull request merged**